✨ NEW: Add file load option for code-cells (#286)
executablebooks/MyST-NB: docs/use/markdown.md

print(f"{a} {b}")
```

**Code from Files:**

`myst_nb` provides a convenience feature for importing executable code into a `{code-cell}`
from a file. This can be useful when you want to share code between documents. To do this

> you specify a `file` metadata attribute such as:

Reviewer: ```suggestion
you specify a `myst_file` metadata attribute such as:
```

````md
```{code-cell} ipython3

> :file: <path>

Reviewer: ```suggestion
:myst_file: <path>
```

Author: @chrisjsewell not sure why you want this to be named `myst_file` as they are files that contain code such as `py` and `jl` files.

Author: I updated the error name from `MystFileParsingErorr` to `CodeFileParsingError` to avoid confusion.

Reviewer: my tentativeness here is that `file` is a pretty generic key and on the off-chance anyone already has this key in their metadata it will now break their builds. Is there perhaps something a little less generic; `import_file` or whatever?

```
````

```{warning}

> This is a feature provided by `myst_nb` and is **not** part of the core `MyST` markup specification.

Reviewer: ```suggestion
This is an experimental feature that is **not** part of the core `MyST` markup specification, and may be removed in the future.
```
